packages/api/src/command/medical/document/document-webhook.ts

```diff
@@ -91,6 +93,18 @@ export const processPatientDocumentRequest = async (
       });
     }
 
+    if (whType === DOWNLOAD_WEBHOOK_TYPE || whType === CONVERSION_WEBHOOK_TYPE) {
```

Comment:
shouldnt this be inside the if statement above? If isWebhookDisabled is true, then this block will still get evaluated right? And then we will be saying we sent webhooks when we really didnt

Reply:
Well no, either way we want to signify whether we have created the webhook request so that way even if we arent sending it to the cx we will only create the request once per ProgressType (download | convert)